wp-content/themes/wp-starter/templates/404.html

```diff
@@ -0,0 +1,14 @@
+<!-- wp:template-part {"slug":"header","area":"header","tagName":"header"} /-->
+
+<!-- wp:group {"tagName":"main","style":{"spacing":{"padding":{"top":"var:preset|spacing|40","bottom":"var:preset|spacing|40"}}},"layout":{"type":"constrained"}} -->
+<main class="wp-block-group" style="padding-top:var(--wp--preset--spacing--40);padding-bottom:var(--wp--preset--spacing--40)">
+	<!-- wp:heading {"level":1} -->
+		<h1 class="wp-block-heading" id="page-not-found">Page Not Found</h1>
+	<!-- /wp:heading -->
+	<!-- wp:paragraph -->
+		<p>The page you are looking for does not exist, or it has been moved.</p>
+	<!-- /wp:paragraph -->
```

Comment:
Tried at first to put this in a pattern, but then ran into errors with it not pulled from that pattern. So, I kept it simple.
It will mostly be updated per site